### ISSUE TITLE
[Gym] - UserWarning (obs returned by the `step()` method is not within the observation space)

### ISSUE THREAD

**Issue by ai-nikolai** [microsoft/TextWorld#324]
When running textworld + gym the following UserWarning is emitted: (when typing `help`)
```bash
lib/python3.9/site-packages/gym/utils/passive_env_checker.py:167: UserWarning: WARN: The obs returned by the `step()` method is not within the observation space with exception: 
  logger.warn(f"{pre} is not within the observation space with exception: {e}")
```

_Software Versions:_
1. textworld==1.5.4
2. numpy==1.26.2
3. gym==0.25.2
4. gym-notices==0.0.8

**Comment by MarcCote** [comment]
Unless you are planning to use off-the-shelf RL algorithm (e.g., [baselines](https://github.com/openai/baselines)), you can safely ignore those warnings.

**Comment by MarcCote** [comment]
Dropping the `gym` dependency.
Fixed by #341
